backend/tests/test_operator_history_payloads.py

```
@@ -184,6 +194,60 @@ def test_operator_history_payloads_can_represent_runtime_control_denials() -> No
+        assert "token" not in serialized
+        assert "secret" not in serialized
+
+
+def test_operator_history_auth_context_rejects_raw_identity_provider_material() -> None:
+    with pytest.raises(ValidationError):
+        OperatorHistoryAuthContext(
+            actor_subject="user:alice",
+            session_id="application-session-redacted",
+            auth_source="enterprise-bridge",
+            entitlement_decision="allow",
+            csrf_token="csrf-token-123",
```

Ruff (0.15.11)

[error] Possible hardcoded password assigned to argument: "csrf_token"

(S106)